test/Chainweb/Test/CutDB.hs

```diff
@@ -476,10 +469,13 @@ fakePact :: WebPactExecutionService
 fakePact = WebPactExecutionService $ PactExecutionService
   { _pactValidateBlock =
       \_ d -> return
-              $ payloadWithOutputs d coinbase $ getFakeOutput <$> _payloadDataTransactions d
+              $ payloadWithOutputs d coinbase
+              $ getFakeOutput <$> _payloadDataTransactions d
   , _pactNewBlock = \_ _ -> do
-        payload <- generate $ V.fromList . getNonEmpty <$> arbitrary
-        return $ newPayloadWithOutputs fakeMiner coinbase payload
+        payloadDat <- generate $ V.fromList . getNonEmpty <$> arbitrary
+        return
+            $ newPayloadWithOutputs fakeMiner coinbase
+            $ (\x -> (x, getFakeOutput x)) <$> payloadDat
```

Comment:
This fixes a bug in `fakePact`. In the old code `_pacValidateBlock` and `_pactNewBlock` generated inconsistent  outputs, which resulted in different block payload hashes and block hashes.